electron-app/src/components/dashboard/SummaryCard.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <v-card
+    :loading="isLoading"
+    :class="`dashboard__summary-card__${name}`"
+    class="pb-3"
+  >
+    <v-row
+      no-gutters
+      class="pa-3 rotkibeige primary--text summary-card__header"
+    >
+      <v-toolbar-title class="font-weight-medium">
+        {{ name }} balances
+        <v-tooltip v-if="this.$slots.tooltip" bottom>
+          <template #activator="{ on }">
+            <v-icon
+              small
+              class="mb-3 ml-1 summary-card__header__tooltip"
+              v-on="on"
+            >
+              fa fa-info-circle
+            </v-icon>
+          </template>
+          <div>
+            <slot name="tooltip"></slot>
+          </div>
+        </v-tooltip>
+      </v-toolbar-title>
+      <v-spacer></v-spacer>
+      <span>
+        <v-tooltip v-if="canRefresh" bottom>
+          <template #activator="{ on }">
+            <v-btn
+              icon
+              x-small
+              color="primary"
+              class="summary-card__refresh-icon"
+              @click="refresh(name)"
+              v-on="on"
+            >
+              <v-icon color="primary">
+                fa-refresh
+              </v-icon>
+            </v-btn>
+          </template>
+          Refresh {{ name }} balances
+        </v-tooltip>
+      </span>
+    </v-row>
+    <v-list>
+      <slot></slot>
+    </v-list>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+@Component({})
+export default class SummaryCard extends Vue {
+  @Prop({ required: true })
+  name!: string;
+  @Prop({ required: false })
+  icon?: string;
```

Comment:
Can we avoid having an undefined property? Vue starts logging warnings for this, when it comes to undefined reactive properties. Better alternative would be to have a default empty string, which is also falsy. Or make it either `string | null` and initialized it as null.

Reply:
Understood. Removing this entirely as it's actually not used/needed anymore. If needed later we can add it back in.